### PR TITLE
Enemy fixes

### PR DESCRIPTION
**Pull request by dariormt**

### CODE REVIEW

**Comment by dariormt** [comment]
Melee weapon support for the enemy is still under development :)

**Comment by ynks** [comment]
do we merge this branches to #92 or to dev directly?

File: Assets/Enemy/EnemyDamageable.cs [hunk 1]
```diff
@@ -1,19 +1,124 @@
+using System;
 using System.Collections;
 using System.Collections.Generic;
+using System.Dynamic;
+using TheKiwiCoder;
+using Unity.VisualScripting;
```
Comment (ynks) [inline Assets/Enemy/EnemyDamageable.cs:6]: jiji

File: Assets/Player/PlayerInput.cs [hunk 2]
```diff
@@ -53,6 +54,10 @@ public void OnEnable()
         _input.Gameplay.Debug.Enable();
         _input.Gameplay.Movement.Enable();
         _input.Gameplay.Aim.Enable();
+        
+        _input.Gameplay.ThrowOrGet.Enable();
+        _input.Gameplay.Fire.Enable();
+        _input.Gameplay.SwitchWeapons.Enable();
```
Comment (ynks) [inline Assets/Player/PlayerInput.cs:60]:
```suggestion
       playerWeaponManager.EnableInput();
```

**Review by ynks — CHANGES_REQUESTED** [review]
Fix the input

File: Assets/Player/PlayerWeaponManager.cs [hunk 3]
```diff
@@ -1,6 +1,7 @@
 using System.Collections.Generic;
 using UnityEngine;
 using UnityEngine.InputSystem;
+using UnityEngine.Windows;
```
Comment (ynks) [inline Assets/Player/PlayerWeaponManager.cs:4]:
```cpp
#using "window.h"
```
🤓

**Review by ynks — APPROVED** [review]

**Pull request merged**